responses/de/HassGetState.yaml

```
@@ -61,3 +61,10 @@ responses:
 
       wie_viele: |
         {{ query.matched | length }}
+
+      wo: |
+        {% if state == "not_home" %}
+          {{ slots.name | capitalize }} ist weg
+        {% else %}
+          {{ slots.name | capitalize }} ist hier: {{ state.state }}
+        {% endif %}
```

```suggestion
      wo: |
         {% if state.state == "not_home" %} 
           {{ slots.name | capitalize }} ist weg
         {% elif state.state == "home" %} 
           {{ slots.name | capitalize }} ist zuhause 
         {% else %} 
           {{ slots.name | capitalize }} ist im Bereich {{ state.state }} 
         {% endif %} 
```

i think the Dutch approach fits better in this case
https://github.com/home-assistant/intents/blob/09ad6c6abbc04671d776bd8e44f1bf5b3eed3280/responses/nl/HassGetState.yaml#L83-L90